src/main/resources/jenkins/security/plugins/ldap/Messages.properties

```diff
@@ -1,6 +1,7 @@
 FromUserRecordLDAPGroupMembershipStrategy.DisplayName=Parse user attribute for list of LDAP groups
 FromGroupSearchLDAPGroupMembershipStrategy.DisplayName=Search for LDAP groups containing user
 LDAPSecurityRealm.AtLeastOne=At least one LDAP server configuration must be configured.
+LDAPSecurityRealm.NotSameServer=Each server configuration must have a unique server string
```

Comment:
As your example above it is easy to have distinguished server strings for the same physical server; just prefix the other one with `ldap://` for example.